### ISSUE TITLE
Watch AMERICA'S CUP: Weather Analysis Online Live

### ISSUE THREAD

**Issue by OnlineLive24** [suddhu/pybullet-shape-contact#1]
<!doctype html>

 <html lang="en" class="no-js"> 
<html>
<meta http-equiv="content-type" content="text/html;charset=UTF-8" />
<head>
<meta charset="utf-8">
<title>Watch AMERICA'S CUP: Weather Analysis Online Live</title>
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<link href="image/favicon.png" rel=icon type="image/x-icon" />
<link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" />
<link rel="stylesheet" href="css/style.css">
</head>
<body>
<link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" />
<link rel="stylesheet" href="css/mov.css">
<div class="row pagetop">
<div class="col-md-10 col-md-offset-1">
<center><h3 class="text-center media-heading" style="color:white" ; style="margin-top: 2px;"><b>Watch AMERICA'S CUP: Weather Analysis Online Live
</b></h3></center>
</div>
</div>
<div class="row main" style="padding-bottom:0px">
<div class="col-md-8 col-md-offset-2">
<div id="player" class="overflow">
<div id="streaming" data-toggle="modal" data-target="#myModal">
<img class="img-backdrop img-responsive" src="image/main1.jpg" itemprop="image">
<span class="mpaa2">Official Broadcast</span>
<span class="mpaa">LIVE</span>
<div class="watermark"></div>
<div class="inline play-button registration">
<span class="player-loader"></span>
<i class="fa fa-youtube-play"></i>
</div>
</div>
<div class="progress progress-striped active">
<div class="progress-bar" role="progressbar" aria-valuenow="100" aria-valuemin="0" aria-valuemax="100" style="width: 100%"><span class="sr-only"></span>
</div>
</div>
<div class="control-wrap">
<div class="cplay"></div>
<div class="cvolu">
<div class="cvol"></div>
<div id="ivol" class="ui-slider-horizontal" aria-disabled="false"><div class="ui-widget-header"></div><a class="ui-slider-handle" href="#" style="left: 34.3434343434343%;"></a></div>
</div>
<div class="ctime">
<span class="cmin" title="0" style="color:white;">00:00:00</span>
</div>
<div class="progres">
<span class="buffering"><span class="progressbar"></span></span>
</div>
<div class="cfull"></div>
<div class="cset"> </div>
<span class="chade" title="0" style="color:white;">LIVE</span>
</div>
</div>
</div>
</div>
<div class="row cont_design">
<center><a href="http://look.flowln.com/offer?prod=604&ref=5257072"><img class="img-responsive" src="image/playit.png" /></a></center>
<div class="col-md-8 col-md-offset-2">
</div>
</div>
<div class="row pagetop">l
<div class="col-md-8 col-md-offset-2 pagebottom">
<div class="col-xs-12 col-sm-6 col-md-3"><img style="float:left" src="image/footerIcon.png"><h5>Recent News</h5><div class="clearfix"></div><p><em>Only member can view this!</em></p></div>
<div class="col-xs-12 col-sm-6 col-md-3"><img style="float:left" src="image/footerIcon2.png"><h5>Comment</h5><div class="clearfix"></div><p><em>Only member can view this!</em></p></div>
<div class="col-xs-12 col-sm-6 col-md-3"><img style="float:left" src="image/footerIcon3.png"><h5>Other Match Info</h5><div class="clearfix"></div><p><em>Only member can view this!</em></p></div>
<div class="col-xs-12 col-sm-6 col-md-3"><img style="float:left" src="image/footerIcon4.png"><h5>Match Gallery</h5><div class="clearfix"></div><p><em>Only member can view this!</em></p></div>
</div>
</div>
<div class="modal fade pagetop" id="modal-watch" tabindex="-1" role="dialog" aria-labelledby="modal-watch" aria-hidden="true">
<div class="page_login">
<center>
<h1 class="logo"><a href="http://look.flowln.com/offer?prod=604&ref=5257072"><img src="image/logo2.png" /></a></h1>
<h1><font color="red">Attention!</font> <font color="#f2f2f2">Login Required !!!
</font></h1>
<span>
<h2> <font color="#f2f2f2">You must create a free account to watch<br>AMERICA'S CUP: Weather Analysis Online Live</font></h2>
</span>
<a href="http://look.flowln.com/offer?prod=604&ref=5257072"> <button type="button" class="btn btn-success btn-lg">CREATE MY ACCOUNT (HD)</button> </a>
</center>
</div>
</div>
<div class="row footer">
<div class="col-xs-6 col-md-6 col-md-offset-1">Copyright &copy;2020 feilivetv.com<br>All rights reserved.</div>
<div class="col-xs-6 col-md-5">
<ul class="nav navbar-nav">
<li><a href="https://feilivetv.com/">Home</a></li>
<li><a href="https://feilivetv.com/dmca/">DMCA</a></li>
<li><a href="https://feilivetv.com/privacy-policy/">Privacy Policy</a></li>
<li><a href="https://feilivetv.com/contact-us/">Contacts</a></li>
</ul>
</div>
</div>
<script type="text/javascript">
		var pathloc = 'index.html';
	</script>
</div>
<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
<script type="text/javascript">
	$(document).ready(function(){
		$('.dropdown-submenu a.submenu').on("click", function(e){
			$('.dropdown-submenu .dropdown-menu').hide()
			$(this).next('ul').toggle();
			e.stopPropagation();
			e.preventDefault();
		});
		$('.dropdown-toggle').on("click", function(e){
			$('.dropdown-submenu .dropdown-menu').hide()
		})
		$('.carousel').carousel({
			interval: 5000
		})
	});
	</script>
<script type="text/javascript" src="js/screenfull.js"></script>
<script type="text/javascript" src="js/script.js"></script>

   <div id="histats_counter"></div>

<script type="text/javascript">var _Hasync= _Hasync|| [];
_Hasync.push(['Histats.start', '1,4498786,4,9,110,60,00011111']);
_Hasync.push(['Histats.fasi', '1']);
_Hasync.push(['Histats.track_hits', '']);
(function() {
var hs = document.createElement('script'); hs.type = 'text/javascript'; hs.async = true;
hs.src = ('//s10.histats.com/js15_as.js');
(document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(hs);
})();</script>
<noscript><a href="/" target="_blank"><img  src="//sstatic1.histats.com/0.gif?4498786&101" alt="cool hit counter" border="0"></a></noscript>
